### PR TITLE
Adding linters

### PR DESCRIPTION
**Pull request by AbuTalha3**
# Setting Linters 
In this project [Hello Microverse](https://github.com/AbuTalha3/Hello-Microverse):
- I used h1 tag for html file that includes a header with the text "Hello Microverse!"
- I created a [gitignore](https://github.com/AbuTalha3/Hello-Microverse/blob/adding%3Dlinters/.gitignore) file and included the node_modules directory and test.md file in it.
- I installed [Stylelint](https://stylelint.io/) and [Webhint](https://webhint.io/) packages to my package json file
- I configured my [linters.yml](https://github.com/AbuTalha3/Hello-Microverse/blob/adding%3Dlinters/.github/workflows/linters.yml), [.hintrc](https://github.com/AbuTalha3/Hello-Microverse/blob/adding%3Dlinters/.hintrc) and [.stylelintrc.json](https://github.com/AbuTalha3/Hello-Microverse/blob/adding%3Dlinters/.stylelintrc.json)

### CODE REVIEW

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @AbuTalha3 , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 

 ## Highlights
- Correct Github flow✔
- Meaningful commit messages👋 ✔

 ## Required Changes ♻️

- [ ] Currently, your pull request does not have a summary description, As part of the requirement for every pull request, you need to add a summary, The summary is a brief point that let you tell others about changes you've pushed to a branch in a repository on GitHub. I'll suggest you click on the 3 dots at the top `see screenshot bellow` and add a proper description for this pull request. :+1: :smile: 
- - Selection_048


> Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- [ ] You did great using the provided Microverse readme template. :+1: :+1: Looking at the readme file, it seems you have not customized it to your project. According to Microverse the only optional section is the FAQ, you are meant to customize every other section in your readme file. :smile: :wave: I'll suggest you go through your readme and remove the placeholder logo and texts, and then fill the content of every section in the readme file :+1: :100: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -245,14 +245,14 @@ I would like to thank...
...
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+<!-- _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._ -->
```
Comment (ademibowale) [inline README.md]: - [ ] According to your previouse reviewer i'll suggest you go through your readme and remove the placeholder texts, and then fill the content of every section in the readme file  to make it more descriptive 👍 💯

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @AbuTalha3  , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [x]  _Please check the comments under the review._

- [ ] According to your previouse reviewer kindly add a descriptive PR summary a brief point that let you tell others about changes you've pushed to a branch in a repository on GitHub eg what you have implemented in this project `hello microverse` example :I Use h1 tag for html file that includes a header with the text "Hello Microverse!".   , 

## Optional suggestions

- [ ]  I would make this as optional suggestion as the previouse code reviewer didnt mentioned this 
 but i would suggest you to use html file that includes a header with the text "Hello Microverse!". Use h1 tag for that according to the requirement.
- [ ] Kindly make your css file  one class with styles for your header

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @AbuTalha3, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**